packages/web/src/public/locales/en/bible.json

```diff
@@ -0,0 +1,250 @@
+{
+  "gen": "Genesis",
+  "gen_short": "Gen",
+  "gen_match": ["Genesis", "Gen", "Gn"],
```

Comment:
I think you are absolutely right about using an algorithm to deal with all the possible variations. Generating all of these is going to be tedious and error prone. Let's tackle that as a separate story